Assets/Scripts/Managers/QuitConfirmationManager.cs

```diff
@@ -0,0 +1,41 @@
+using UI;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.Serialization;
+
+public class QuitConfirmationManager : MonoBehaviour
+{
+    //Access to the QuitConfirmation panel
+    [FormerlySerializedAs("confirmationPanel")] [SerializeField] GameObject confirmationCanvas;
+    //Access to main menu scene name
+    [SerializeField] private string mainMenuScene = "MainMenu";
```

Comment:
my scene list manager should've been merged so we can use that now:
`SceneListManager.Instance.LoadMainMenu()`